packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -25,6 +25,24 @@ import 'tooltip_visibility.dart';
 /// Signature for when a tooltip is triggered.
 typedef TooltipTriggeredCallback = void Function();
 
+/// Signature for computing the position of a tooltip.
+///
+/// The arguments are:
+/// * `target`: The offset of the target the tooltip is positioned near in the global coordinate system.
```

Comment:
Is this offset the center of the target? If so, we might want to clarify it in documentation.